pennylane/operation.py

```diff
@@ -2096,6 +2094,14 @@ def queue(self, context=QueuingManager, init=False):  # pylint: disable=argument
 
             context.remove(o)
 
+        wires = [op.wires for op in self.obs]
+        if len(wires) != len(set(wires)):
+            warnings.warn(
+                "Tensor object acts on overlapping wires; in some PennyLane functions this will "
+                "lead to undefined behaviour",
```

Comment:
What does this mean?

Reply:
This warning is raised when someone does something like `qml.PauliX(0) @ qml.PauliZ(0)`. `Tensor` isn't supposed to have observables with overlapping wires as it was never intended for that. Although this warning isn't new, I just relocated it :)

Reply:
Thanks! I'm just thinking of what I'd do with this warning... and wondering why it's not an error.